GDSCommon-Demo/GDSCommon-Demo/Mocks/MockErrorViewModel.swift

```diff
@@ -0,0 +1,16 @@
+import GDSCommon
+import UIKit
+
+struct MockErrorViewModel: GDSErrorViewModel, BaseViewModel {
+    var image: String = "exclamationmark.circle"
+    var title: GDSLocalisedString = "This is an Error View title"
+    var body: GDSLocalisedString = "This is an Error View body This is an Error View body"
+    var primaryButtonViewModel: ButtonViewModel = MockButtonViewModel.primary
+    var secondaryButtonViewModel: ButtonViewModel? = MockButtonViewModel.secondary
+    var rightBarButtonTitle: GDSLocalisedString?
+    var backButtonIsHidden: Bool = false
+    
+    func didAppear() {}
+    
+    func didDismiss() {}
```

Comment:
If you do as @amopatel1234 is suggesting and use this mock instead of the test view model, you'll need to move the action closures over to this view model and call them in these methods :)